x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_view.tsx

```diff
@@ -0,0 +1,178 @@
+  const { navigateToUrl } = useValues(KibanaLogic);
+  const collectionViewTabs = [
+    {
+      id: 'events',
+      label: 'Events',
+      onClick: () =>
+        navigateToUrl(
+          generateEncodedPath(COLLECTION_VIEW_PATH, {
+            name: analyticsCollection.name,
+            section: 'events',
+          })
+        ),
+      isSelected: section === 'events',
+    },
+    {
+      id: 'integrate',
+      label: 'Integrate',
+      prepend: <EuiIcon type="editorCodeBlock" size="l" />,
+      onClick: () =>
+        navigateToUrl(
+          generateEncodedPath(COLLECTION_VIEW_PATH, {
+            name: analyticsCollection.name,
+            section: 'integrate',
+          })
+        ),
+      isSelected: section === 'integrate',
+    },
+    {
+      id: 'settings',
+      label: 'Settings',
+      onClick: () =>
+        navigateToUrl(
+          generateEncodedPath(COLLECTION_VIEW_PATH, {
+            name: analyticsCollection.name,
+            section: 'settings',
+          })
+        ),
+      isSelected: section === 'settings',
+    },
+  ];
+
+  useEffect(() => {
+    fetchAnalyticsCollection(name);
+  }, []);
+
+  return (
+    <EnterpriseSearchAnalyticsPageTemplate
+      restrictWidth
+      isLoading={isLoading}
+      pageChrome={[...collectionViewBreadcrumbs]}
+      pageViewTelemetry="View Analytics Collection"
+      pageHeader={{
+        description: i18n.translate(
+          'xpack.enterpriseSearch.analytics.collections.pageDescription',
+          {
+            defaultMessage:
+              'Dashboards and tools for visualizing end-user behavior and measuring the performance of your search applications. Track trends over time, identify and investigate anomalies, and make optimizations.',
+          }
+        ),
+        pageTitle: i18n.translate('xpack.enterpriseSearch.analytics.collections.pageTitle', {
+          defaultMessage: 'Behaviorial Analytics',
+        }),
+        tabs: [...collectionViewTabs],
+      }}
+    >
+      {hasNoAnalyticsCollection && (
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiTitle>
+              <h2>
+                {i18n.translate('xpack.enterpriseSearch.analytics.collections.headingTitle', {
+                  defaultMessage: 'Collections',
+                })}
+              </h2>
+            </EuiTitle>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiButtonTo fill iconType="plusInCircle" to={COLLECTION_CREATION_PATH}>
+              {i18n.translate('xpack.enterpriseSearch.analytics.collections.create.buttonTitle', {
+                defaultMessage: 'Create new collection',
+              })}
+            </EuiButtonTo>
+          </EuiFlexItem>
+        </EuiFlexGroup>
+      )}
+
+      <EuiSpacer size="l" />
+      {hasNoAnalyticsCollection ? (
+        <EuiEmptyPrompt
+          iconType="search"
+          title={
+            <h2>
+              {i18n.translate(
```

Comment:
Have a 404 message here instead. "You may have deleted this analytics collection"